doc/whats_new/v0.24.rst

```diff
@@ -12,6 +12,16 @@ Version 0.24.2
 Changelog
 ---------
 
+:mod:`sklearn.gaussian_process`
+...............................
+
+- |Fix| Avoid explicitly forming inverse covariance matrix in
```

Comment:
move this entry in l. 48 in the already existing `sklearn.gaussian_process` section.